src/lib/qnio/nio_server.c

```diff
@@ -412,6 +430,17 @@ qns_server_start(char *node, char *port)
                         nioDbg("Accepted connection on descriptor %d "
                                   "(host=%s, port=%s)\n", infd, hbuf, sbuf);
                     }
+                    if(cmn_ctx->ssl_ctx)
+                    {
+                        ssl = SSL_new(cmn_ctx->ssl_ctx);
+                        SSL_set_fd(ssl, infd);
+                        ret = SSL_accept(ssl);
```

Comment:
Correctness isn't a "feature".  If you don't want to write asynchronous code then don't use an event loop architecture.  Even if Veritas doesn't use this library for the server side, you are providing this reference implementation and it should do things properly.  It's bad for VxHS if people get the impression that corners were cut and things weren't done properly.